[Bug]: Confusing panic when config.toml for the node is invalid

### Problem encountered

When the below section is missing in the node config.toml We are getting panic that says nothing for less technical people(or the one who does not know vega code base well)

```
  [[EvtForward.EVMBridges]]
    Level = "Info"
    MaxEthereumBlocks = 500
    PollEventRetryDuration = "500ms"
    ChainID = "421614"
    SkipClientVerification = true
    HeartbeatIntervalForTestOnlyDoNotChange = "0s"
```


### Observed behaviour

Panic happens instead of human readable error

### Expected behaviour

Error should be reported with information what is really wrong

### Steps to reproduce

```bash
1. Setup validator
2. Remove the `[[EvtForward.EVMBridges]]` section from the `config.toml`
```


### Software version

v0.78.X

### Failing test

_No response_

### Jenkins run

_No response_

### Configuration used

_No response_

### Relevant log output

```shell
panic: runtime error: index out of range [0] with length 0
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x3361962]

goroutine 1 [running]:
code.vegaprotocol.io/vega/core/protocol.New.func1()
        /home/runner/work/vega/vega/core/protocol/protocol.go:81 +0x1a2
panic({0x3c0e1c0?, 0xc0003b2d98?})
        /opt/hostedtoolcache/go/1.21.13/x64/src/runtime/panic.go:920 +0x270
code.vegaprotocol.io/vega/core/protocol.newServices({0x4871a70?, 0xc000bfa7d0}, 0xc011322ae0, 0xc000c1e900, 0xc000df9bf0, 0xc0001342d0, 0xc000134370, 0xc0005c5400, 0xc000704980, 0xc00024eec0, ...)
        /home/runner/work/vega/vega/core/protocol/all_services.go:261 +0x7046
code.vegaprotocol.io/vega/core/protocol.New({0x4871a70, 0xc000bfa7d0}, 0xc000c1e900, 0x3?, 0x1?, 0xbbb?, 0xc000d52b10?, 0x7?, 0x101?, 0xc0005c5400, ...)
        /home/runner/work/vega/vega/core/protocol/protocol.go:87 +0x1a9
code.vegaprotocol.io/vega/cmd/vega/commands/node.(*Command).Run(0xc001062000, 0xc000c1e900, {0x48974e8?, 0xc000d6c010}, {0xc001052458, 0x6}, {0x7ffe3994319f, 0x2e}, {0x7ffe399431dc, 0x5d}, ...)
        /home/runner/work/vega/vega/cmd/vega/commands/node/node.go:125 +0x54c
code.vegaprotocol.io/vega/cmd/vega/commands.(*StartCmd).Execute(0x6deb780, {0x3bbe200, 0x6deb780, 0xb})
        /home/runner/work/vega/vega/cmd/vega/commands/node.go:118 +0x80e
github.com/jessevdk/go-flags.(*Parser).ParseArgs(0xc000c04c60, {0xc000218130?, 0x5, 0x5})
        /home/runner/go/pkg/mod/github.com/jessevdk/go-flags@v1.4.0/parser.go:316 +0x86f
github.com/jessevdk/go-flags.(*Parser).Parse(...)
        /home/runner/go/pkg/mod/github.com/jessevdk/go-flags@v1.4.0/parser.go:186
code.vegaprotocol.io/vega/cmd/vega/commands.Main({0x4871a00, 0x6e22ca0})
        /home/runner/work/vega/vega/cmd/vega/commands/root.go:92 +0x538
main.main()
        /home/runner/work/vega/vega/cmd/vega/main.go:27 +0x25
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -20,6 +20,7 @@
 
 - [11672](https://github.com/vegaprotocol/vega/issues/11672) - Add missing fees in GraphQL bindings.
 - [11681](https://github.com/vegaprotocol/vega/issues/11681) - Account for conflicts inserting funding payment records.
+- [11684](https://github.com/vegaprotocol/vega/issues/11684) - Better error when `Arbitrum` bridge details are missing from validator configuration.
 - [11696](https://github.com/vegaprotocol/vega/issues/11696) - Add binding for estimate fees API.
 
 

--- a/core/protocol/all_services.go
+++ b/core/protocol/all_services.go
@@ -255,6 +255,10 @@ func newServices(
 	svcs.forwarderHeartbeat = evtforward.NewTracker(log, svcs.witness, svcs.timeService)
 
 	if svcs.conf.HaveEthClient() {
+		if len(svcs.conf.EvtForward.EVMBridges) != 1 {
+			return nil, fmt.Errorf("require exactly 1 [[EvtForward.EVMBridges]] in configuration file, got: %d", len(svcs.conf.EvtForward.EVMBridges))
+		}
+
 		svcs.primaryBridgeView = bridges.NewERC20LogicView(primaryEthClient, primaryEthConfirmations)
 		svcs.secondaryBridgeView = bridges.NewERC20LogicView(secondaryEthClient, secondaryEthConfirmations)
 		svcs.primaryEventForwarderEngine = evtforward.NewEngine(svcs.log, svcs.conf.EvtForward.Ethereum)

